spdk.spec

```diff
@@ -175,15 +169,16 @@ mv doc/output/html/ %{install_docdir}
 
 
 %files
-%{_bindir}/spdk_*
 %dir %{_datadir}/%{name}
 %{_libdir}/*.so.*
+%exclude %{_libdir}/pkgconfig
```

Comment:
This is superfluous.  You are not adding anything to this package's `%files` that would include `pkgconfig/`